website/src/controllers/api-v3/groups.js

```diff
@@ -308,6 +308,57 @@ api.joinGroup = {
 };
 
 /**
+ * @api {post} /groups/:groupId/reject Reject a group invitation
+ * @apiVersion 3.0.0
+ * @apiName RejectGroupInivite
+ * @apiGroup Group
+ *
+ * @apiParam {UUID} groupId The group _id
+ *
+ * @apiSuccess {Object} group The group
+ */
+api.rejectGroupInivite = {
+  method: 'POST',
+  url: '/groups/:groupId/reject-invite',
+  middlewares: [authWithHeaders(), cron],
+  async handler (req, res) {
+    let user = res.locals.user;
+
+    req.checkParams('groupId', res.t('groupIdRequired')).notEmpty().isUUID();
+
+    let validationErrors = req.validationErrors();
+    if (validationErrors) throw validationErrors;
+
+    let group = await Group.getGroup({user, groupId: req.params.groupId, optionalMembership: true}); // Do not fetch chat and work even if the user is not yet a member of the group
```

Comment:
you're right, let's not fetch the group at all and return an empty object. I'm not even sure if this route belongs in the groups controller, what about moving it to the user controller?